src/composables/multisig.ts

```diff
@@ -13,12 +31,35 @@ interface UseMultisigOptions {
   store: Store<any>
 }
 
-/**
- * This composable provides methods related to multisig operations
- */
-export function useMultisig({ store }: UseMultisigOptions) {
+const POLLING_INTERVAL = 5000;
+const MULTISIG_TRANSCATION_EXPARATION_HEIGHT = 50;
```

Comment:
EXPARATION => EXPIRATION
TRANSCATION => TRANSACTION

@CedrikNikita I suggest to install this VSC extension: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker